### DISCUSSION TITLE
firmware version v1.2

### DISCUSSION THREAD

**Discussion by hbarthel** [phiresky/ebv-theta-to-mqtt#1] [Q&A]
My heating control Theta v1.2 is from 2002 something. Is there a chance to read out data? From what I understood the protocol on my T2B bus is different.
Cheers, Heiko

**Comment by phiresky** [comment]
I'm not sure.. What's the exact product line and product name? Do you have
a link to the installation manual?

**Reply by hbarthel** [reply]
It's a Theta N 23B V1.2
https://www.regotherm24.de/media/pdf/29/3d/23/0450010000_0811-30_BA_THETA_NURS_EbV_DE.pdf
But my manual has number 045 001 0000 - 0325-11
All in German.

**Reply by phiresky** [reply]
So that's the same device I have just a different (firmware?) revision. I doubt the protocol is very different between revisions, but really the only way to find out is to try it since there is no official documentation about these things anywhere. You just need a laptop and a USB-to-TTL adapter to try it which are pretty cheap.

Also, what I'm using is The RS-485 interface, which is a separate thing from the T2B bus (I think.. not sure)
